src/main/java/uk/ac/cam/cl/dtg/isaac/dos/Hub.java

```diff
@@ -0,0 +1,25 @@
+package uk.ac.cam.cl.dtg.isaac.dos;
```

Comment:
I've just reviewed the front end changes. It seems like we have the same data duplicated front and back (hub enums). Worth considering whether to have a single source for this data on the backend & pull it on the frontend as required.

Reply:
Types are generated by maven plugin and fed into the front end and content editor type files, hence it looks like duplication.